services/client/src/components/video/icon.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const getSourceIcon = source => {
+  if (source === "youtube")
+    return "https://upload.wikimedia.org/wikipedia/commons/thumb/7/72/YouTube_social_white_square_%282017%29.svg/2000px-YouTube_social_white_square_%282017%29.svg.png";
```

Comment:
❓ Could we use the fontawesome youtube icon? https://fontawesome.com/icons/youtube?style=brands

Reply:
Yes; I assume it has icons for all the other major providers so I'll give it a go

Reply:
The default font awesome icon for youtube isn't great; the one above is available in the fontawesome brands but it feels like overkill to include all brands for 1-2 icons (Youtube, and Meetup). Eventbrite, NISF, and NIDC, are not present. Instead, I've created the https://github.com/apoclyps/my-dev-space/issues/227 issue to create a sprite sheet with the icons and replace the icons once we have a definitive list.